youtube_dl/extractor/hketv.py

```diff
@@ -0,0 +1,157 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..compat import compat_str
+from ..utils import (
+    ExtractorError,
```

Comment:
Alphabetic.

Reply:
Thanks.  I thought it were supposed to be in ASCII order, i.e. upper-case first, but then checking other extractors reveals that my assumption was wrong.  Fixed.